File: NET/com.yupanastudio.igest.winapp.model/admin/Opcion.cs
using System.Collections.Generic;

namespace com.yupanastudio.igest.winapp
{
    public class Opcion
    {
        public int idOpcion { get; set; }
        public int idPadre { get; set; }
        public string idSistema { get; set; }
        public int idPerfil { get; set; }
        public string nomOpcion { get; set; }
        public string nomControl { get; set; }
        public int orden { get; set; }
        public int idTipoOpcion { get; set; }
        public int idTipoEmpresa { get; set; }
        public bool activo { get; set; }
        public bool premium { get; set; }
        public int idEmpresa { get; set; }

        public bool soloLectura { get; set; }       //BORRAR LUEGO de quitar todas las referncias

        public string permisos { get; set; }

        public List<CrudDB> lsPermisos { get; set; }

        public bool ver { get; set; }
        public bool crear { get; set; }
        public bool editar { get; set; }
        public bool eliminar { get; set; }
        public bool imprimir { get; set; }


    }//end class

}

File: SPRING/igest.server.model/src/main/java/com/yupanastudio/igest/server/principal/dml/Opcion.java
/*
 * Document   : Opcion
 * Author     : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.dml;

public class Opcion {

    private int idOpcion;
    private int idPadre;
    private String idSistema;
    private int idPerfil;
    private String nomOpcion;
    private String nomControl;
    private int orden;
    private int idTipoOpcion;
    private int idTipoEmpresa;
    private Boolean activo;
    private Boolean premium;
    private int idEmpresa;

    public int getIdOpcion() {
        return idOpcion;
    }

    public void setIdOpcion(int idOpcion) {
        this.idOpcion = idOpcion;
    }

    public int getIdPadre() {
        return idPadre;
    }

    public void setIdPadre(int idPadre) {
        this.idPadre = idPadre;
    }

    public String getIdSistema() {
        return idSistema;
    }

    public void setIdSistema(String idSistema) {
        this.idSistema = idSistema;
    }

    public int getIdPerfil() {
        return idPerfil;
    }

    public void setIdPerfil(int idPerfil) {
        this.idPerfil = idPerfil;
    }

    public String getNomOpcion() {
        return nomOpcion;
    }

    public void setNomOpcion(String nomOpcion) {
        this.nomOpcion = nomOpcion;
    }

    public String getNomControl() {
        return nomControl;
    }

    public void setNomControl(String nomControl) {
        this.nomControl = nomControl;
    }

    public int getOrden() {
        return orden;
    }

    public void setOrden(int orden) {
        this.orden = orden;
    }

    public int getIdTipoOpcion() {
        return idTipoOpcion;
    }

    public void setIdTipoOpcion(int idTipoOpcion) {
        this.idTipoOpcion = idTipoOpcion;
    }

    public int getIdTipoEmpresa() {
        return idTipoEmpresa;
    }

    public void setIdTipoEmpresa(int idTipoEmpresa) {
        this.idTipoEmpresa = idTipoEmpresa;
    }

    public Boolean getActivo() {
        return activo;
    }

    public void setActivo(Boolean activo) {
        this.activo = activo;
    }

    public Boolean getPremium() {
        return premium;
    }

    public void setPremium(Boolean premium) {
        this.premium = premium;
    }

    public int getIdEmpresa() {
        return idEmpresa;
    }

    public void setIdEmpresa(int idEmpresa) {
        this.idEmpresa = idEmpresa;
    }

}
